proposals/0034-glulx-random.md

```diff
@@ -81,7 +81,12 @@ Considerations:
     call is safe in affected interpreters, or instead whether it should make a
     ranged call using a high prime number, or if any priming is necessary.
 
-3. We need to replace the Inform 6 `random` function rather than introducing a
+3. Unless the new RNG is in predictable mode, it will need to be reseeded from
+    `@random` after restoring or undoing. This means an extra variable will be
+    needed to record whether the RNG is in 'random' or 'predictable' mode, as
+    the RNG's own state would otherwise be indistinguishable.

```

Comment:
What does "predictable mode" mean in this context? It's already a somewhat fuzzy notion to me when talking about Glulx terps and their `{random,setrandom}` opcodes, but it's even less clear to me what it's supposed to mean for kit code. I would have expected that this RNG is just reseeded via the terp's `@random` on every restore/restart. Why is that not sufficient? What problem do you want to address with this "RNG mode" business?

Reply:
Predictable mode refers to seeding the RNG with a specified seed. That allows, for example, everyone to play the same randomly generated dungeon in Kerkerkruip.